charts/civil-general-applications/values.preview.template.yaml

```diff
@@ -212,13 +212,7 @@ ccd:
         ROLE_ASSIGNMENT_URL: http://${SERVICE_NAME}-am-role-assignment-service
```

Comment:
this needs to be changed to aat am role assignment as well since we have deleted the pod

Reply:
Done. Thank you